### ISSUE TITLE
Fix Extension for High Contrast Theme

### ISSUE THREAD

**Issue by Tanya0609** [Microsoft/WebTemplateStudio#1096]
Sync with @xtinah-w, some UI in the high contrast theme mode are not showing properly. In High Contrast, everything should be dark background and blue boarder highlight.

<img width="1280" alt="Screen Shot 2019-08-20 at 10 23 00 PM" src="https://user-images.githubusercontent.com/12874317/63404856-a0ff2880-c399-11e9-8a19-4382248de2c7.png">
<img width="1234" alt="Screen Shot 2019-08-20 at 10 23 43 PM" src="https://user-images.githubusercontent.com/12874317/63404859-a2305580-c399-11e9-9cfc-6ef6a0769e91.png">
<img width="354" alt="Screen Shot 2019-08-20 at 10 24 10 PM" src="https://user-images.githubusercontent.com/12874317/63404862-a3618280-c399-11e9-9809-6ca263cdc4c2.png">
<img width="1235" alt="Screen Shot 2019-08-20 at 10 25 04 PM" src="https://user-images.githubusercontent.com/12874317/63404865-a492af80-c399-11e9-88f8-406a6692dc63.png">
<img width="1235" alt="Screen Shot 2019-08-20 at 10 26 01 PM" src="https://user-images.githubusercontent.com/12874317/63404869-a5c3dc80-c399-11e9-8150-89a1e1716147.png">

**Comment by Tanya0609** [comment]
[docs of overwrite vs code colors](https://code.visualstudio.com/api/references/contribution-points#contributes.colors) 

[docs editor widget colors](https://code.visualstudio.com/api/references/theme-color#editor-widget-colors)

[docs editor colors](https://code.visualstudio.com/api/references/theme-color#editor-colors)
